server/sonar-server/src/test/java/org/sonar/server/projecttag/ws/SearchActionTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.projecttag.ws;
+
+import java.io.IOException;
+import org.junit.Test;
+import org.sonar.api.server.ws.WebService;
+import org.sonar.server.measure.index.ProjectMeasuresIndex;
+import org.sonar.server.ws.WsActionTester;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.sonar.test.JsonAssert.assertJson;
+
+public class SearchActionTest {
+
+  private ProjectMeasuresIndex index = mock(ProjectMeasuresIndex.class);
+
+  private WsActionTester ws = new WsActionTester(new SearchAction(index));
+
+  @Test
+  public void json_example() throws IOException {
+    when(index.searchTags(anyString(), anyInt())).thenReturn(newArrayList("official", "offshore", "playoff"));
+
+    String result = ws.newRequest().execute().getInput();
+
+    assertJson(ws.getDef().responseExampleAsString()).withStrictArrayOrder().isSimilarTo(result);
+  }
+
```

Comment:
WDYM? Isn't it tested in ProjectMeasuresIndex? I don't want to test the same thing twice